src/Audio.js

```diff
@@ -83,16 +83,19 @@ function Audio() {
 
     return (
         <div className="audioContainer">
+            <div style={{width: "100%"}}>
             < WavesurferPlayer
                 height={100}
-                waveColor="blue"
+                waveColor="red"
                 url={testAudio}
                 onReady={onReady}
                 onPlay={() => setIsPlaying(true)
                 }
                 onPause={() => setIsPlaying(false)}
+                barWidth={4}
+                barRadius={50}
```

Comment:
bar width makes less granular bars and radius makes the edges round. These can be tweaked but this looks nicer